chainstate/types/src/error.rs

```diff
@@ -32,6 +32,8 @@ pub enum PropertyQueryError {
     BlockNotFound(Id<Block>),
     #[error("Previous block index not found {0}")]
     PrevBlockIndexNotFound(Id<GenBlock>),
+    #[error("Block index at height not found {0}")]
```

Comment:
```suggestion
    #[error("Block index at height {0} not found")]
```

incredibly picky but I feel this would be neater